docs/references/database/list-documents.md

```diff
@@ -1 +1 @@
-Get a list of all the user documents. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's documents. [Learn more about different API modes](/docs/admin).
\ No newline at end of file
+Get a list of all the user's documents. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's documents. [Learn more about different API modes](/docs/admin).
```

Comment:
as i think now this is misleading. even on admin mode, the endpoint returns list of all of the documents in the `collection` and `database` provided, not the project right?

Reply:
Maybe: `Get a list of all the user's documents in the given database.`

Reply:
Maybe: Get a list of all the user's documents in the given collection.